packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -46,6 +47,9 @@ abstract class RenderSliverFixedExtentBoxAdaptor extends RenderSliverMultiBoxAda
   /// The main-axis extent of each item.
   double get itemExtent;
 
+  /// The main-axis extent builder of each item.
+  ItemExtentGetter? get itemExtentBuilder => null;
```

Comment:
What if when null, this builder just returns itemExtent for every index by default? Maybe it could simplify the code below with `if (itemExtentBuilder == null)`?

Reply:
I have added some docs for this.
```dart
  /// The main-axis extent builder of each item.
  ///
  /// If this is non-null, the [itemExtent] will be ignored.
  /// If this is null, the [itemExtent] must be positive finite.
  ItemExtentBuilder? get itemExtentBuilder => null;
```